src/toil/common.py

```diff
@@ -56,6 +56,7 @@ def __init__(self):
         #Batch system options
         self.batchSystem = "singleMachine"
         self.scale = 1
+        self.mesosCredentials = '/etc/mesos/credentials.txt'
```

Comment:
cgcloud has it as `.txt`. I'll change it there as well.

refer https://github.com/BD2KGenomics/cgcloud/pull/109